cedar-lean/Cedar/Spec/Response.lean

```diff
@@ -32,7 +32,8 @@ inductive Decision where
 
 structure Response where
   decision : Decision
-  policies : Set PolicyID
+  determiningPolicies : Set PolicyID
+  erroredPolicies : Set PolicyID
```

Comment:
Nit: I have a slight preference for `erroringPolicies` or `errorPolicies`.

Reply:
`errorPolicies` conflicts with the function name in `Authorizer.lean`, but I like `erroringPolicies` for symmetry-ish with `determiningPolicies`